lxd/endpoints/network_util.go

```diff
@@ -0,0 +1,49 @@
+package endpoints
+
+import (
+	"net"
+	"regexp"
+
+	"github.com/lxc/lxd/shared/logger"
+)
+
+type networkServerErrorLogWriter struct {
+	proxies []net.IP
+}
+
+// Regex for the log we want to ignore.
+var unwantedLogRegex = regexp.MustCompile(`http: TLS handshake error from ([^\[:]*|\[(.*?)]):.*write: connection reset by peer$`)
+
+func (d networkServerErrorLogWriter) Write(p []byte) (int, error) {
+	if d.shouldDiscard(p) {
+		return 0, nil
+	}
+
+	logger.Info(string(p))
```

Comment:
It's occurred to me that this will duplicate the time and date on the log.

Reply:
I've changed this to write to stderr. If you want to normalise all of the logs and use `logger.Info` we would probably need to do more regexing to trim the prefix that is added by `log.Logger`.